ci/check-version-file.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Perhaps a brief description at the top of the file about what this script intends to do.